source/server/http/config_tracker_impl.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <envoy/server/config_tracker.h>
+
+#include "common/common/assert.h"
+#include "common/common/macros.h"
+
+namespace Envoy {
+namespace Server {
+
+/*
+ * Implementation of ConfigTracker
```

Comment:
nit: period end of sentence. Same elsewhere.

Reply:
kk